pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -137,6 +137,11 @@ self: super: {
     then addBuildDepend super.halive pkgs.darwin.apple_sdk.frameworks.AppKit
     else super.halive;
 
+  # Test suite fails due golden tests checking text representation
+  # of normalized dhall expressions, and newer dhall versions format
+  # differently.
+  hpack-dhall = dontCheck super.hpack-dhall;
```

Comment:
Is this something you could report upstream?

If so, please create an issue upstream, and link to it here in this commit right above the `hpack-dhall = ` line.

We like to be able to have some way of tracking when these uses of `dontCheck` can be removed.

Reply:
It is already in master, but not in release: https://github.com/cabalism/hpack-dhall/commit/5049aa4dd9dc79b41d11e2c72dffddd9bc92b203

Reply:
Suggestion: Can you add an assert on hpack-dhall.version with a comment to remove the dontCheck, when the assert fails?